assets/stylesheets/layout/_main.scss

```diff
@@ -541,3 +541,8 @@ body.newdash div.wordpress-com-extension-promo {
 		display: none;
 	}
 }
+
+//added to the html element when we don't want the background content to scroll
+.noscroll {
```

Comment:
I'd add the `html` selector just for safety, and maybe adjust the class to be `no-scroll`? So `html.no-scroll`.